api/controller/psmdb_cluster.proto

```diff
@@ -64,6 +66,8 @@ message PSMDBClusterParams {
   ];
   // Docker image used for PSMDB.
   string image = 3;
+  // Version service URL.
+  string version_service_url = 4;
```

Comment:
I don't think that it's a good idea to pass version_service_url in PSMDB cluster URL. If we need it in UI it's better to return it in settings API.